src/toolbar.scss

```diff
@@ -140,64 +141,64 @@ $title-toolbar-height: 2.75rem;
     background-color: var(--sapToolbar_SeparatorColor);
   }
 
-  &__overflow {
-    &.#{$fd-namespace}-popover {
-      .#{$block}__overflow__body {
-        @include fd-reset();
-
-        display: flex;
-        flex-direction: column;
-        max-width: 20rem;
-        padding: 0.1875rem 0.375rem;
-
-        > * {
-          display: block;
-          text-align: left;
-          width: auto;
-
-          @include fd-rtl() {
-            text-align: right;
-          }
-        }
-        > .#{$block}__separator {
-          height: 0.0625rem;
-          margin: 0.375rem 0.1875rem;
-          width: auto;
-          background-color: var(--sapToolbar_SeparatorColor);
-        }
-
-        .#{$block}__overflow__label {
-          &.#{$fd-namespace}-label {
-            margin: 0.375rem 0;
-          }
-        }
-
-        .#{$block}__overflow__form-label {
-          &.#{$fd-namespace}-form-label {
-            margin: 0.625rem 0 0.125rem 0;
-          }
-        }
-        .#{$block}__overflow__form-label,
-        .#{$block}__overflow__label {
-          overflow: hidden;
-          white-space: nowrap;
-          text-overflow: ellipsis;
-        }
+  &__overflow__body {
+    @include fd-reset();
+
+    display: flex;
+    flex-direction: column;
+    max-width: 20rem;
+    padding: 0.1875rem 0.375rem;
+
+    > *,
+    > .#{$fd-namespace}-button {
```

Comment:
This needs to be removed - there should be no universal selectors (`*`) ever and we should not be using `.#{$fd-namespace}-*` as discussed previously.